includes/admin/class-wc-admin-setup-wizard.php

```diff
@@ -747,74 +930,102 @@ protected function get_wizard_payment_gateways() {
-						<div class="wc-wizard-gateway-enable">
-							<input type="checkbox" name="wc-wizard-gateway-<?php echo esc_attr( $gateway_id ); ?>-enabled" class="input-checkbox" value="yes" />
-							<label>
-								<?php if ( $gateway['image'] ) : ?>
-									<img src="<?php echo esc_attr( $gateway['image'] ); ?>" alt="<?php echo esc_attr( $gateway['name'] ); ?>" />
-								<?php else : ?>
-									<?php echo esc_html( $gateway['name'] ); ?>
-								<?php endif; ?>
-							</label>
-						</div>
-						<div class="wc-wizard-gateway-description">
-							<?php echo wp_kses_post( wpautop( $gateway['description'] ) ); ?>
-						</div>
-						<?php if ( ! empty( $gateway['settings'] ) ) : ?>
-							<table class="form-table wc-wizard-gateway-settings">
-								<?php foreach ( $gateway['settings'] as $setting_id => $setting ) : ?>
-									<tr>
-										<th scope="row"><label for="<?php echo esc_attr( $gateway_id ); ?>_<?php echo esc_attr( $setting_id ); ?>"><?php echo esc_html( $setting['label'] ); ?>:</label></th>
-										<td>
-											<input
-												type="<?php echo esc_attr( $setting['type'] ); ?>"
-												id="<?php echo esc_attr( $gateway_id ); ?>_<?php echo esc_attr( $setting_id ); ?>"
-												name="<?php echo esc_attr( $gateway_id ); ?>_<?php echo esc_attr( $setting_id ); ?>"
-												class="input-text"
-												value="<?php echo esc_attr( $setting['value'] ); ?>"
-												placeholder="<?php echo esc_attr( $setting['placeholder'] ); ?>"
-												/>
-										</td>
-									</tr>
-								<?php endforeach; ?>
-							</table>
-						<?php endif; ?>
-					</li>
-				<?php endforeach; ?>
+			<?php if ( $can_stripe ) : ?>
+				<p><?php esc_html_e( 'Your store will be set up to accept payments instantly on checkout with Stripe.', 'woocommerce' ); ?></p>
+			<?php else : ?>
+				<p><?php printf( __( 'WooCommerce can accept both online and offline payments. <a href="%1$s" target="_blank">Additional payment methods</a> can be installed later and managed from the <a href="%2$s" target="_blank">checkout settings</a> screen.', 'woocommerce' ), esc_url( admin_url( 'admin.php?page=wc-addons&view=payment-gateways' ) ), esc_url( admin_url( 'admin.php?page=wc-settings&tab=checkout' ) ) ); ?></p>
+			<?php endif; ?>
+			<ul class="wc-wizard-services featured">
+				<?php foreach ( $featured_gateways as $gateway_id => $gateway ) :
+					$this->display_service_item( $gateway_id, $gateway );
+				endforeach; ?>
+			</ul>
+			<ul class="wc-wizard-services in-cart">
+				<?php foreach ( $in_cart_gateways as $gateway_id => $gateway ) :
+					$this->display_service_item( $gateway_id, $gateway );
+				endforeach; ?>
+			</ul>
+			<ul class="wc-wizard-services manual">
+				<li class="wc-wizard-services-list-toggle">
+					<div class="wc-wizard-service-name">Manual Payments</div>
+					<div class="wc-wizard-service-description">
+						Collect payments from customers outside your online store.
```

Comment:
Missing translations.

Reply:
Fixed in https://github.com/woocommerce/woocommerce/pull/16755/commits/74d1b36f425e8a61fd8e09da5fa2fa358718f152.